sunbird_mcp/src/api/sandbox_search/validation.py

```
@@ -0,0 +1,151 @@
+            validated['offset'] = offset
+        except (ValueError, TypeError):
+            errors.append("Offset must be a non-negative integer")
+    
+    # Validate fields and facets
+    fields = params.get('fields', [])
+    facets = params.get('facets', [])
+    if fields or facets:
+        errors.extend(validate_fields_and_facets(fields, facets))
+    
+    if 'fields' in params:
+        if isinstance(params['fields'], list) and all(isinstance(x, str) for x in params['fields']):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)